packages/mc-html-template/html-docs/application.html

```diff
@@ -166,9 +166,7 @@
     __LOADING_SCREEN_JS__
 
     <!-- Application globals -->
-    <script>
-      window.app = __APPLICATION_ENVIRONMENT__;
-    </script>
+    __APPLICATION_ENVIRONMENT__
```

Comment:
This now ensures that prettier doesn't mess with the format

Reply:
Wow. That's quite a find.